listenbrainz/webserver/views/dump_api.py

```diff
@@ -0,0 +1,62 @@
+from flask import Blueprint, request, jsonify
+from listenbrainz.webserver.errors import APIBadRequest, APINotFound
+from listenbrainz.webserver.rate_limiter import ratelimit
+
+import listenbrainz.db.dump as db_dump
+
+dump_api_bp = Blueprint("dump_api_v1", __name__)
+
+
+@dump_api_bp.route("/get-info", methods=["GET"])
+@ratelimit()
+def get_info():
+    """
+    Get information about ListenBrainz data dumps.
+    You need to pass the `id` parameter in a GET request to get data about that particular
+    dump.
+
+    **Example response**:
+
+    .. sourcecode:: json
+        {
+            "id": 1,
+            "timestamp": "20190625-165900"
+        }
+
+    :query id: *Required.* Integer specifying the ID of the dump
```

Comment:
What if I don't have one? If not specified, I think we should return the date and id of the latest dump...

Reply:
I was returning 400 if there wasn't an ID provided, I've changed it now to return the latest.